program/src/main.rs

```diff
@@ -94,7 +70,7 @@ pub fn main() {
     let head = store.finalized_header.beacon().slot;
 
     let proof_outputs = ProofOutputs {
-        executionStateRoot: execution_state_proof.execution_state_root,
+        executionStateRoot: *store.finalized_header.execution().unwrap().state_root(),
```

Comment:
I would unwrap this in a safer way. E.g. If the unwrap fails, panic with `execution payload doesn't exist`